packages/pwa/app/commerce-api/auth.js

```diff
@@ -136,6 +104,8 @@ class Auth {
             return this._pendingLogin
         }
 
+        this._api.shopperLogin.clientConfig.throwOnBadResponse = true
```

Comment:
I'm assuming that with the addition of this flag we now have the opportunity to clean up some of our exception handling code throughout the commerce api folder?